src/metrics_to_extract.py

```diff
@@ -220,150 +220,150 @@
 }
 
 bamstats_metrics = {
-    'Estimate_Average_Coverage': {
-        'key': 'Average coverage (estimated)',
-        'tooltip': 'Estimated average coverage',
-        'derived_from': 'bamstats:estimate_average_coverage',
-        'type': float
-    },
+    # 'Estimate_Average_Coverage': {
```

Comment:
Let's remove this for now. Picard is calculating this as well but the values are slightly different (therefore it's confusing). I would trust the Picard value more.

Reply:
agreed! did you take it away already?

Reply:
I commented it out